onadata/apps/api/viewsets/xform_list_viewset.py

```diff
@@ -44,8 +44,10 @@ class XFormListViewSet(ETagsMixin, BaseViewset,
     content_negotiation_class = MediaFileContentNegotiation
     filter_backends = (filters.XFormListObjectPermissionFilter,
                        filters.XFormListXFormPKFilter)
-    queryset = XForm.objects.filter(downloadable=True, deleted_at=None,
-                                    is_merged_dataset=False)
+    queryset = XForm.objects.filter(
+        downloadable=True, deleted_at=None,
+        is_merged_dataset=False).only('id_string', 'title', 'version', 'uuid',
+                                      'description', 'user__username')
```

Comment:
You should add the field `hash` here and make the relevant change in the `XFormListSerializer`.